services/generic/alarm.md

```diff
@@ -1,41 +1,80 @@
+| `alarm_siren`       | inactive, siren_active                                                                                                                                                                                                                                                                   |
+| `alarm_system`      | hw_failure, sw_failure, hw_failure_with_code, sw_failure_with_code                                                                                                                                                                                                                       |
+| `alarm_time`        | wakeup, timer_ended, time_remaining                                                                                                                                                                                                                                                      |
+| `alarm_water_valve` | valve_op, master_valve_op, valve_short_circuit, current_alarm, alarm_water_valve, master_valve_current_alarm                                                                                                                                                                             |
+| `alarm_water`       | leak, level_drop, replace_filter                                                                                                                                                                                                                                                         |
+| `alarm_weather`     | inactive, moisture                                                                                                                                                                                                                                                                       |
+
+## Interfaces
+
+| Type | Interface            | Value type | Storage | Description                                                                                                 |
+|------|----------------------|------------|---------|-------------------------------------------------------------------------------------------------------------|
+| in   | cmd.alarm.get_report | str_map    |         | Requests [`event`](#service-names) status. See [`event_request`](#definitions) definition for more details. |
+| out  | evt.alarm.report     | str_map    | `event` | Reports [`event`](#service-names) status. See [`event_report`](#definitions) definition for more details.   |
+| in   | cmd.alarm.clear      | string     |         | Clears [`event`](#service-names) provided in the value.                                                     |
+
+> Interface cmd.alarm.get_report is used in Z-Wave to clear last alarm for a given event.
```

Comment:
Fix this note, based on above comment.

Reply:
Added as agreed "May not be supported by all devices.".